source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRenderer.mm

```diff
@@ -185,7 +224,8 @@ + (UIView *)render:(UIView<ACRIContentHoldingView> *)view
 
     for (const auto &elem : elems) {
         ACRSeparator *separator = nil;
-        if (*firstelem != elem && renderedView && elem->MeetsTargetWidthRequirement(hostWidth)) {
+        BOOL isGridView = ([view isKindOfClass:[ARCGridViewLayout class]]);
```

Comment:
I will recommend calling new Method similar to renderInFlow and call that from all container.  Instead of patching this method for AreaGrid or flow. Two main reasons:

1. It gives clear separation b/w old vs new layout logic. Which can be extended to newer layout systems.
2. This method assumes that layout is based on stack. Handles visibility differently.